src/twisted/internet/test/test_inlinecb.py

```diff
@@ -189,13 +189,9 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         tb = f.getTraceback()
-        self.assertIn("in erroring", tb)
         self.assertIn("in calling", tb)
-        self.assertIn("in calling2", tb)
-        self.assertIn("in calling3", tb)
-        self.assertNotIn("throwExceptionIntoGenerator", tb)
+        self.assertIn("calling2", tb)
```

Comment:
I didn't had time to check what is going on here... but at first view, this test change doesn't look ok.

It just removes the check...and is similar to not having a test at all.

---------

If it's easier to implement, we can have 2 separate tests.
One for 3.13 and newer and another for 3.12 and older.

Reply:
I've split these out into Python 3.13 and Python 3.12 and below.

Reply:
> Still looking into why this check is saying my tests aren't covered by tests.

Thanks for the tests.

On Python 3.13 we don't have test coverage reporting.

Last time I tried, `coverage.py` was not yet supported on Python 3.13 https://github.com/nedbat/coveragepy/issues/1682

Reply:
also... [slipcover](https://github.com/plasma-umass/slipcover/issues/51) doesn't yet support Python 3.13 ... so for 3.13 we are left uncovered :)